swagger.yaml

```diff
@@ -467,6 +467,46 @@ paths:
           description: "Internal error occurred"
           schema:
             $ref: '#/definitions/ErrorResponse'
+    put:
+      tags:
+        - Groups
+      summary: "Update an existing group"
+      description: "Update an existing group in Cognito user pool"
+      consumes:
+        - application/json
+      produces:
+        - "application/json"
+      parameters:
+        - in: body
+          name: "Group"
+          description: "The updated details of the group"
+          schema:
+            required: ["name", "precedence"]
```

Comment:
Should this not be 'description'?

Reply:
Changed this to `name` as per ticket https://trello.com/c/i7MqRBIg/670-670-add-human-readable-group-name:

`API Req -> Cognito - > API resp
name -> Description -> name`